Makefile

```diff
@@ -213,17 +227,76 @@ update-codegen:
 	$(call run-informer-gen,)
 .PHONY: update-codegen
 
+# $1 - target file
+define generate-crd
+	$(CONTROLLER_GEN) crd paths="$(GO_PACKAGES)" output:crd:stdout > '$1'
+endef
+
+update-crd:
+	$(call generate-crd,pkg/api/scylla/v1/scylla.scylladb.com_scyllaclusters.yaml)
```

Comment:
nit: the local path seem pretty prone to typos, might worth to make it a private variable